src/runtime/server/lib/oidc.ts

```diff
@@ -156,15 +157,15 @@ export function callbackEventHandler({ onSuccess, onError }: OAuthConfig<UserSes
         {
           method: 'POST',
           headers,
-          body: config.tokenRequestType === 'json' ? requestBody : requestForm
+          body: convertTokenRequestToType(requestBody, config.tokenRequestType)
         }
       )
     } catch (error: any) {
       // Log ofetch error data to console
-      logger.error(error.data)
+      logger.error(error?.data ?? error)
 
       // Handle Microsoft consent_required error
-      if (error.data.suberror === 'consent_required') {
+      if (error?.data?.suberror === 'consent_required') {
```

Comment:
This fixes issues I ran into where I was getting normal request errors without a `data` property but the true error was obscured by the ReferenceError of trying to read the undefined data.